src/test/components/FlameGraph.test.js

```diff
@@ -67,17 +68,12 @@ describe('FlameGraph', function () {
     expect(drawCalls).toMatchSnapshot();
   });
 
-  it('renders a message instead of the graph when call stack is inverted', () => {
-    const { getByText, dispatch } = setupFlameGraph();
-    dispatch(changeInvertCallstack(true));
-    expect(getByText(/The Flame Graph is not available/)).toBeInTheDocument();
-  });
```

Comment:
Even if you don't want to do the cleanup now, it would be good to at least leave a TODO to the MaybeFlamegraph component for future readers wondering about the existing useless code.

Reply:
Added a commit which adds a comment.